site/templates/reference-block.php

```diff
@@ -0,0 +1,31 @@
+<?php snippet('cheatsheet.article.header') ?>
+
+<div class="text">
+  <h2 id="preview"><a href="#preview">Preview</a></h2>
+  <?= kirbytag('screenshot', $page->image()->filename()) ?>
+
+  <?= $page->text()->kt() ?>
+
+  <h2 id="default-files"><a href="#default-files">Default files</a></h2>
+
+  <h3 id="default-files__snippet"><a href="#default-files__snippet">Snippet</a></h3>
+  <?= kt('
+```php "/site/snippets/blocks/' . $page->slug() . '.php"
```

Comment:
I'd remove the file location, because the default file doesn't exist there (there is also no location given for the blueprint).